packages/core/src/config/models.ts

```diff
@@ -23,153 +23,174 @@ export type DeepPartial<T> = {
 export type SitecoreConfigInput = {
   /**
    * API settings required to connect to Sitecore.
-   * Both edge and local set can be specified as Content SDK app will use API Key for component library
+   * Both edge and local sets can be specified; the Content SDK app will choose
+   * the correct credentials (Edge or local) at runtime.
    */
   api?: {
     /**
-     * Edge endpoint credentials for Sitecore connection. Will be used to connect to SaaS XMCloud instance
+     * Edge endpoint credentials for connecting to an XM Cloud instance.
      */
     edge?: {
-      // for now contextID will take the role of both server and client IDs
       /**
        * A unified identifier used to connect and retrieve data from XM Cloud instance
+       * Must be provided together with `clientContextId` to support both server-
+       * side and browser-side data fetching.
        */
       contextId: string;
-      // clientContextId will be utilized when we know more specifics about it
+
       /**
        * Optional identifier used to connect and retrieve data from XM Cloud instance in client-side functionality
        */
       clientContextId?: string;
+
       /**
        * XM Cloud endpoint that the app will communicate and retrieve data from
        * @default https://edge-platform.sitecorecloud.io
        */
       edgeUrl?: string;
     };
+
     /**
-     * API endpoint credentials for connection to local Sitecore instance
+     * API endpoint credentials for connecting to a local Sitecore instance.
      */
     local?: {
       /**
-       * Sitecore API key identifier used to connect to the GraphQL endpoint
+       * Sitecore API key used to connect to the GraphQL endpoint
        */
       apiKey: string;
+
       /**
-       * Sitecore API hostname that the app will connect and retrieve data from
+       * Sitecore API hostname that the app connects to
        */
       apiHost: string;
+
       /**
-       * GraphQL endpoint path, will be appended to apiHost to form full enpoint URL ($apiHost/$path)
+       * GraphQL endpoint path (appended to `apiHost` to form the full URL).
        * @default /sitecore/api/graph/edge
        */
       path?: string;
     };
   };
+
   /**
    * The default and fallback locale for your site.
    * Ensure it aligns with the framework-specific settings used in your application.
    */
   defaultLanguage?: string;
+
   /**
-   * Your default site name. When using the multisite feature this variable defines the fallback site.
+   * Default site name. When using multisite, this is the fallback site.
```

Comment:
Please retain `this variable defines the fallback site` wording, as our multisite logic is somewhat specific like that